odoo/addons/test_new_api/tests/test_one2many.py

```diff
@@ -327,3 +329,21 @@ def test_parent_id(self):
         # Check that the search is not stuck in the loop
         Team.search([('id', 'parent_of', team1.id)])
         Team.search([('id', 'child_of', team1.id)])
+
+    def test_new_one2many_compute_batching(self):
```

Comment:
This issue actually has nothing to do with one2many fields.
It's only a computed field on a recordset, isn't it?

Reply:
You are perfectly right, I changed it.